std/testing/asserts.ts

```diff
@@ -412,7 +412,7 @@ export function assertMatch(
 /**
  * Forcefully throws a failed assertion
  */
-export function fail(msg?: string): void {
```

Comment:
Seems like unrelated change. Please revert

Reply:
You're right, thanks! Reverted.